File: GarageGo-BE/Models/DichVu.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GarageGo_BE.Models
{
    [Table("DICHVU")]
    public class DichVu
    {
        [Key]
        [Column("MaDV")]
        public int Id { get; set; }

        [Required]
        [StringLength(100)]
        [Column("TenDV")]
        public string TenDichVu { get; set; } = string.Empty;

        [StringLength(500)]
        public string? MoTa { get; set; }

        [StringLength(255)]
        [Column("HinhAnh")]
        public string? HinhAnh { get; set; }

        [Required]
        [Column("DonGia", TypeName = "decimal(18,2)")]
        public decimal Gia { get; set; }

        [Column("TrangThai")]
        public bool TrangThai { get; set; } = true;
    }
}

File: GarageGo-BEJava/src/main/java/com/garagego/model/DichVu.java
package com.garagego.model;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "DICHVU")
public class DichVu {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "MaDV")
    private Integer maDV;

    @Column(name = "TenDV", nullable = false, length = 100)
    private String tenDichVu;

    @Column(name = "MoTa", length = 500)
    private String moTa;

    @Column(name = "HinhAnh", length = 255)
    private String hinhAnh;

    @Column(name = "DonGia", nullable = false, precision = 18, scale = 2)
    private BigDecimal gia;

    @Column(name = "TrangThai")
    private Boolean trangThai = true;
}
